modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/RelatedObjectEntryResourceTest.java

```diff
@@ -107,47 +108,203 @@ public void setUp() throws Exception {
 			_systemObjectDefinitionMetadataTracker.
 				getSystemObjectDefinitionMetadata("User");
 
-		ObjectDefinition userSystemObjectDefinition =
+		_userSystemObjectDefinition =
 			_objectDefinitionLocalService.fetchSystemObjectDefinition(
 				_userSystemObjectDefinitionMetadata.getName());
 
-		_objectRelationship =
```

Comment:
~~I don't get this refactor.
Why create and populate a relationship in each test instead of doing it once in the setup?
Furthermore, you are creating exactly the same objectRelationship in each test, so, why not use the one defined in the setup?~~
My bad, I realized that you are creating relationships with differents types